bin/node-template/LICENSE

```diff
@@ -1,24 +1,16 @@
-This is free and unencumbered software released into the public domain.
+MIT No Attribution
 
-Anyone is free to copy, modify, publish, use, compile, sell, or
-distribute this software, either in source code form or as a compiled
-binary, for any purpose, commercial or non-commercial, and by any
-means.
+Copyright 2023 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
Copyright Parity Technologies (UK) Ltd.
```
We don't use copyright years anymore - there is no gain and its annoying to maintain.